### PR TITLE
value types, featured projects, and unknowns

### PR DESCRIPTION
**Pull request by ethn**
This is to finish and unify the issues EYSS was working on.

### CODE REVIEW

File: db/migrate_cards/20180921200422_convert_currency_to_unit.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- encoding : utf-8 -*-
+
+class ConvertCurrencyToUnit < Card::Migration
+  UNITS = { /^\$$/ => "USD", /US Dollars/ => "USD" }
+
+  def up
+    Card.search type: "Metric", right_plus: "currency" do |metric|
+      if metric.value_type_code == :money
```
Comment (xithan) [inline db/migrate_cards/20180921200422_convert_currency_to_unit.rb:8]: looks like a `monetary?` method is missing

File: script/csv_import/metric/metric_csv_row.rb [hunk 2]
```diff
@@ -37,10 +37,8 @@ def normalize_value_type value
     value.match(/(?<type>[^(]+)\((?<options>[^)]+)/) do |match|
       new_value = match[:type].strip
       new_value = "Category" if new_value == "Categorical"
-      if new_value == "Category" || new_value == "Multi-Category"
+      if %w[Category Multi-Category].include? new_value
```
Comment (xithan) [inline script/csv_import/metric/metric_csv_row.rb]: on my hit list this is the least preferable option. Maybe we can agree on `new_value.in? %w[Category Multi-Category]`?

File: mod/metrics/lib/javascript/script_metric_properties.js.coffee [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+showPropsFor= (scope, value_type) ->
+  propertiesForValueType(value_type).forEach (prop) ->
+    rowForProp(scope, prop).show()
+
+rowForProp = (scope, prop) ->
+  set = scope.find('.RIGHT-' + prop)
+  if set.closest(METRIC_PROPERTIES_TABLE)[0]
+    set.closest('tr')
+  else
+    set
```
Comment (xithan) [inline mod/metrics/lib/javascript/script_metric_properties.js.coffee:61]: Didn't we want to start teststing js?

File: mod/metrics/spec/lib/formula/javascript/script_value_type.test.coffee.off [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#$.extend global, require('../../../../lib/javascript/script_value_type.js.coffee')
+
+#test 'return true if it returns an array with the properties corresponding to "Number" and "Money"', =>
+#  expect(getPropertiesValueType('Number')).toEqual(['Unit','Range'])
+#  expect(getPropertiesValueType('Money')).toEqual(['Unit','Range'])
+
+#test 'return true if it returns an array with the properties corresponding to "Category" and "Multi-Category"', =>
+#  expect(getPropertiesValueType('Category')).toEqual(['Options'])
+#  expect(getPropertiesValueType('Multi-Category')).toEqual(['Options'])
```
Comment (xithan) [inline mod/metrics/spec/lib/formula/javascript/script_value_type.test.coffee.off:9]: Why is this not active?

**Review by xithan — APPROVED** [review]

File: mod/metrics/spec/lib/formula/javascript/script_value_type.test.coffee.off (hunk 4 above)
Comment (ethn) [reply mod/metrics/spec/lib/formula/javascript/script_value_type.test.coffee.off:9]: I'm  not sure.  Daniel wrote it and then commented it.

File: mod/metrics/lib/javascript/script_metric_properties.js.coffee (hunk 3 above)
Comment (ethn) [reply mod/metrics/lib/javascript/script_metric_properties.js.coffee:61]: we did.  this is perhaps a job for cucumber, though.

**Pull request merged**